packages/lit-dev-content/src/components/algolia-search-controller.ts

```diff
@@ -63,10 +71,15 @@ export class AgloliaSearchController<T extends {}> {
     if (trimmedQuery.length < 2) {
       return [];
     }
-    const results = await this._index.search<T>(trimmedQuery, {
+    type SearchOptions = Parameters<typeof this._index.search>[1];
```

Comment:
Do we need this type? Can the literal just be inlined or the type inferred?

Reply:
Sorry I merged before this comment, but without the type, it won't cause a build failure

Reply:
Actually it won't cause a build failure with the type either, but you get autocomplete to help you out there